packages/react-magma-dom/src/components/Dropdown/DropdownSplitButton.tsx

```diff
@@ -44,9 +44,9 @@ export const DropdownSplitButton = React.forwardRef<
 
   const buttonIcon =
     context.dropDirection === DropdownDropDirection.up ? (
-      <ArrowDropUpIcon size={10} testId="caretUp" />
+      <ArrowDropUpIcon size={24} testId="caretUp" />
```

Comment:
I know you're going to do more changes to the split button in another PR, but we can get the sizes from the theme -- `theme.iconSizes.medium`